visidata/path.py

```diff
@@ -178,6 +178,16 @@ def exists(self):
     def scheme(self):
         if self.is_url():
             return urlparse(self.given).scheme
+        else:
+            error('trying to urlparse a non-url')
+
+    @property
```

Comment:
Let's not commit to a `Path.urlparse`property in the API right now. Maybe we instead adjust the `openurl_` API to take a `url` in the first place.  For this commit let's inline urlparse in the postgres loader and we can revisit later.